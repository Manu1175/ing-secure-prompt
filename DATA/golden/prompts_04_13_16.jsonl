{"id": "prompts_04_13_16:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create an on-call roster for the customer outreach platform with primary Mary Sims (+32 42 361 650, mary.sims@ing.com) and backup John Graham (+32 31 603 900, john.graham@ing.com). Include CorpKeys NV26ZI and DI91EX.", "expected_scrub": "Create an on-call roster for the customer outreach platform with primary <EMP1_FIRST> <EMP1_LAST> (<EMP1_PHONE>, <EMP1_EMAIL>) and backup <EMP2_FIRST> <EMP2_LAST> (<EMP2_PHONE>, <EMP2_EMAIL>). Include CorpKeys <EMP1_CORPKEY> and <EMP2_CORPKEY>.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a contact card for ‘Customer Data Steward’ Erin Johnson (erin.johnson@ing.com, +32 92 249 506) and ‘Privacy Officer’ Elizabeth Sullivan (elizabeth.sullivan@ing.com, +32 71 848 879).", "expected_scrub": "Draft a contact card for roles with <EMP_A_FIRST> <EMP_A_LAST> (<EMP_A_EMAIL>, <EMP_A_PHONE>) and <EMP_B_FIRST> <EMP_B_LAST> (<EMP_B_EMAIL>, <EMP_B_PHONE>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Send a training invite for consent handling to Heather Johnson (NS99DB) and Nicholas Wolf (LW24YO).", "expected_scrub": "Send a training invite for consent handling to <EMP1_FIRST> <EMP1_LAST> (<CORPKEY_1>) and <EMP2_FIRST> <EMP2_LAST> (<CORPKEY_2>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an incident paging cascade SMS: page Daniel Taylor (+32 94 953 438); if no ack in 10 minutes, page Miguel Fitzgerald (+32 81 609 125).", "expected_scrub": "Write an incident paging cascade SMS: page <EMP1_NAME> (<EMP1_PHONE>); if no ack in 10 minutes, page <EMP2_NAME> (<EMP2_PHONE>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a RACI note: Owner Mary Sims; Approver John Graham; Contributors Erin Johnson and Kristy Peterson for customer communications templates.", "expected_scrub": "Create a RACI note: Owner <OWNER_NAME>; Approver <APPROVER_NAME>; Contributors <CONTRIB_1> and <CONTRIB_2> for <WORKSTREAM>.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a profile completeness check for Emily Jones (Bruges; emily.jones@skynet.be; +32 15 726 931). Ask to confirm postal address and marketing preferences.", "expected_scrub": "Write a profile completeness check for <FIRST_NAME> <LAST_NAME> (<CITY>; <EMAIL>; <PHONE>). Ask to confirm postal address and marketing preferences.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a consent & preferences audit note for Emily Smith (emily.smith@telenet.be): verify channels against profile (Location Mons; Interest Finance; Hobbies Traveling).", "expected_scrub": "Compose a consent & preferences audit note for <FIRST_NAME> <LAST_NAME> (<EMAIL>): verify channels against profile (Location <CITY>; Interest <INTEREST>; Hobbies <HOBBIES>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a data-quality ticket: Michael Williams shows Gender 'Female' and Age 27; confirm encoding and correct any inconsistencies.", "expected_scrub": "Draft a data-quality ticket: <EXAMPLE_NAME> shows <FIELD>=<VALUE> and <FIELD_2>=<VALUE_2>; confirm encoding and correct inconsistencies.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a high-income microsegment (≥ €90k): Katie Lewis (Bruges; €95,193), Jane Miller (Ghent; €93,522), Robert Lewis (Ghent; €99,675).", "expected_scrub": "Create a high-income microsegment (≥ <THRESHOLD>): <NAME_1> (<CITY_1>; <INCOME_1>), <NAME_2> (<CITY_2>; <INCOME_2>), <NAME_3> (<CITY_3>; <INCOME_3>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose churn‑risk nudges for Brussels customers: Katie Taylor (katie.taylor@telenet.be), John Johnson (john.johnson@proximus.be), Linda Jones (linda.jones@telenet.be).", "expected_scrub": "Compose churn‑risk nudges for <CITY> customers: <NAME_1> (<EMAIL_1>), <NAME_2> (<EMAIL_2>), <NAME_3> (<EMAIL_3>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a privacy snippet explaining why we collect IP Address and Interest fields and how customers can manage preferences in the portal.", "expected_scrub": "Write a privacy snippet explaining why we collect <FIELD_A> and <FIELD_B> and how customers can manage preferences in the portal.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an engagement snapshot for Emily Jones: Mobile Banking (User, Active; last 2025-04-25) and Current Account (Holder, Active; last 2024-11-11).", "expected_scrub": "Write an engagement snapshot for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>; last <DATE_1>) and <PRODUCT_2> (<RELATIONSHIP>, <STATUS>; last <DATE_2>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a relationship matrix for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).", "expected_scrub": "Create a relationship matrix for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>), <PRODUCT_2> (<RELATIONSHIP>, <STATUS>), <PRODUCT_3> (<RELATIONSHIP>, <STATUS>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Produce a microsegment for Mortgage borrowers (Active): John Lewis (last 2025-09-09), Jane Smith (last 2025-06-08), Tom Smith (last 2025-01-20). Suggest a rate alert campaign.", "expected_scrub": "Produce a microsegment for <PRODUCT> borrowers (<STATUS>): <FULLNAME_1> (last <DATE_1>), <FULLNAME_2> (last <DATE_2>), <FULLNAME_3> (last <DATE_3>). Suggest a campaign.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a re‑engagement nudge for Online Banking users now Inactive: John Taylor (last 2025-03-19) and Katie Taylor (last 2025-05-04).", "expected_scrub": "Draft a re‑engagement nudge for <PRODUCT> users now <STATUS>: <FULLNAME_1> (last <DATE_1>) and <FULLNAME_2> (last <DATE_2>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Cross‑sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Offer a mortgage health check and rate simulation.", "expected_scrub": "Cross‑sell note for <FIRST_NAME> <LAST_NAME>: <PROD_A> (<REL_A>, <STATUS_A>), <PROD_B> (<REL_B>, <STATUS_B>), <PROD_C> (<REL_C>, <STATUS_C>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a consent/cohort note for Emily Smith: Online Banking (User, Active; last 2025-08-07), Investment Portfolio (Investor, Active; last 2025-07-09), Personal Loan (Borrower, Active; last 2024-10-19).", "expected_scrub": "Compose a consent/cohort note for <FIRST_NAME> <LAST_NAME>: <PROD_1> (<REL_1>, <STATUS_1>; last <DATE_1>), <PROD_2> (<REL_2>, <STATUS_2>; last <DATE_2>), <PROD_3> (<REL_3>, <STATUS_3>; last <DATE_3>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data‑minimization note: for customer analytics exports (names, emails, phones, addresses, IDs, DOB, IPs, income), use pseudonymous IDs, mask sensitive fields, and restrict raw access to least‑privileged roles.", "expected_scrub": "Create a data‑minimization note: for customer analytics exports (e.g., <FIELD_LIST>), use pseudonymous IDs, mask sensitive fields, and restrict access.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data‑minimization note: when sharing employee contacts (phones, emails, CorpKeys) for operational use, limit distribution to least‑privileged DLs and label ‘Internal‑Only’.", "expected_scrub": "Create a data‑minimization note: when sharing employee contacts (phones, emails, IDs), limit distribution to least‑privileged groups and label internal‑only.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create an on-call roster for the customer outreach platform with primary Mary Sims (+32 42 361 650, mary.sims@ing.com) and backup John Graham (+32 31 603 900, john.graham@ing.com). Include CorpKeys NV26ZI and DI91EX.", "expected_scrub": "Create an on-call roster for the customer outreach platform with primary <EMP1_FIRST> <EMP1_LAST> (<EMP1_PHONE>, <EMP1_EMAIL>) and backup <EMP2_FIRST> <EMP2_LAST> (<EMP2_PHONE>, <EMP2_EMAIL>). Include CorpKeys <EMP1_CORPKEY> and <EMP2_CORPKEY>.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a contact card for ‘Customer Data Steward’ Erin Johnson (erin.johnson@ing.com, +32 92 249 506) and ‘Privacy Officer’ Elizabeth Sullivan (elizabeth.sullivan@ing.com, +32 71 848 879).", "expected_scrub": "Draft a contact card for roles with <EMP_A_FIRST> <EMP_A_LAST> (<EMP_A_EMAIL>, <EMP_A_PHONE>) and <EMP_B_FIRST> <EMP_B_LAST> (<EMP_B_EMAIL>, <EMP_B_PHONE>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Send a training invite for consent handling to Heather Johnson (NS99DB) and Nicholas Wolf (LW24YO).", "expected_scrub": "Send a training invite for consent handling to <EMP1_FIRST> <EMP1_LAST> (<CORPKEY_1>) and <EMP2_FIRST> <EMP2_LAST> (<CORPKEY_2>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an incident paging cascade SMS: page Daniel Taylor (+32 94 953 438); if no ack in 10 minutes, page Miguel Fitzgerald (+32 81 609 125).", "expected_scrub": "Write an incident paging cascade SMS: page <EMP1_NAME> (<EMP1_PHONE>); if no ack in 10 minutes, page <EMP2_NAME> (<EMP2_PHONE>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a RACI note: Owner Mary Sims; Approver John Graham; Contributors Erin Johnson and Kristy Peterson for customer communications templates.", "expected_scrub": "Create a RACI note: Owner <OWNER_NAME>; Approver <APPROVER_NAME>; Contributors <CONTRIB_1> and <CONTRIB_2> for <WORKSTREAM>.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a profile completeness check for Emily Jones (Bruges; emily.jones@skynet.be; +32 15 726 931). Ask to confirm postal address and marketing preferences.", "expected_scrub": "Write a profile completeness check for <FIRST_NAME> <LAST_NAME> (<CITY>; <EMAIL>; <PHONE>). Ask to confirm postal address and marketing preferences.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a consent & preferences audit note for Emily Smith (emily.smith@telenet.be): verify channels against profile (Location Mons; Interest Finance; Hobbies Traveling).", "expected_scrub": "Compose a consent & preferences audit note for <FIRST_NAME> <LAST_NAME> (<EMAIL>): verify channels against profile (Location <CITY>; Interest <INTEREST>; Hobbies <HOBBIES>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a data-quality ticket: Michael Williams shows Gender 'Female' and Age 27; confirm encoding and correct any inconsistencies.", "expected_scrub": "Draft a data-quality ticket: <EXAMPLE_NAME> shows <FIELD>=<VALUE> and <FIELD_2>=<VALUE_2>; confirm encoding and correct inconsistencies.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a high-income microsegment (≥ €90k): Katie Lewis (Bruges; €95,193), Jane Miller (Ghent; €93,522), Robert Lewis (Ghent; €99,675).", "expected_scrub": "Create a high-income microsegment (≥ <THRESHOLD>): <NAME_1> (<CITY_1>; <INCOME_1>), <NAME_2> (<CITY_2>; <INCOME_2>), <NAME_3> (<CITY_3>; <INCOME_3>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose churn‑risk nudges for Brussels customers: Katie Taylor (katie.taylor@telenet.be), John Johnson (john.johnson@proximus.be), Linda Jones (linda.jones@telenet.be).", "expected_scrub": "Compose churn‑risk nudges for <CITY> customers: <NAME_1> (<EMAIL_1>), <NAME_2> (<EMAIL_2>), <NAME_3> (<EMAIL_3>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a privacy snippet explaining why we collect IP Address and Interest fields and how customers can manage preferences in the portal.", "expected_scrub": "Write a privacy snippet explaining why we collect <FIELD_A> and <FIELD_B> and how customers can manage preferences in the portal.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an engagement snapshot for Emily Jones: Mobile Banking (User, Active; last 2025-04-25) and Current Account (Holder, Active; last 2024-11-11).", "expected_scrub": "Write an engagement snapshot for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>; last <DATE_1>) and <PRODUCT_2> (<RELATIONSHIP>, <STATUS>; last <DATE_2>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a relationship matrix for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).", "expected_scrub": "Create a relationship matrix for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>), <PRODUCT_2> (<RELATIONSHIP>, <STATUS>), <PRODUCT_3> (<RELATIONSHIP>, <STATUS>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Produce a microsegment for Mortgage borrowers (Active): John Lewis (last 2025-09-09), Jane Smith (last 2025-06-08), Tom Smith (last 2025-01-20). Suggest a rate alert campaign.", "expected_scrub": "Produce a microsegment for <PRODUCT> borrowers (<STATUS>): <FULLNAME_1> (last <DATE_1>), <FULLNAME_2> (last <DATE_2>), <FULLNAME_3> (last <DATE_3>). Suggest a campaign.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a re‑engagement nudge for Online Banking users now Inactive: John Taylor (last 2025-03-19) and Katie Taylor (last 2025-05-04).", "expected_scrub": "Draft a re‑engagement nudge for <PRODUCT> users now <STATUS>: <FULLNAME_1> (last <DATE_1>) and <FULLNAME_2> (last <DATE_2>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Cross‑sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Offer a mortgage health check and rate simulation.", "expected_scrub": "Cross‑sell note for <FIRST_NAME> <LAST_NAME>: <PROD_A> (<REL_A>, <STATUS_A>), <PROD_B> (<REL_B>, <STATUS_B>), <PROD_C> (<REL_C>, <STATUS_C>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a consent/cohort note for Emily Smith: Online Banking (User, Active; last 2025-08-07), Investment Portfolio (Investor, Active; last 2025-07-09), Personal Loan (Borrower, Active; last 2024-10-19).", "expected_scrub": "Compose a consent/cohort note for <FIRST_NAME> <LAST_NAME>: <PROD_1> (<REL_1>, <STATUS_1>; last <DATE_1>), <PROD_2> (<REL_2>, <STATUS_2>; last <DATE_2>), <PROD_3> (<REL_3>, <STATUS_3>; last <DATE_3>).", "entities": null, "notes": null}
{"id": "prompts_04_13_16:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data‑minimization note: for customer analytics exports (names, emails, phones, addresses, IDs, DOB, IPs, income), use pseudonymous IDs, mask sensitive fields, and restrict raw access to least‑privileged roles.", "expected_scrub": "Create a data‑minimization note: for customer analytics exports (e.g., <FIELD_LIST>), use pseudonymous IDs, mask sensitive fields, and restrict access.", "entities": null, "notes": null}
{"id": "prompts_04_13_16:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_13_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data‑minimization note: when sharing employee contacts (phones, emails, CorpKeys) for operational use, limit distribution to least‑privileged DLs and label ‘Internal‑Only’.", "expected_scrub": "Create a data‑minimization note: when sharing employee contacts (phones, emails, IDs), limit distribution to least‑privileged groups and label internal‑only.", "entities": null, "notes": null}
