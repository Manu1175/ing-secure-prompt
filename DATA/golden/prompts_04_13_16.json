[
  {
    "text": "Create an on-call roster for the customer outreach platform with primary Mary Sims (+32 42 361 650, mary.sims@ing.com) and backup John Graham (+32 31 603 900, john.graham@ing.com). Include CorpKeys NV26ZI and DI91EX.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Draft a contact card for ‘Customer Data Steward’ Erin Johnson (erin.johnson@ing.com, +32 92 249 506) and ‘Privacy Officer’ Elizabeth Sullivan (elizabeth.sullivan@ing.com, +32 71 848 879).",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Send a training invite for consent handling to Heather Johnson (NS99DB) and Nicholas Wolf (LW24YO).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write an incident paging cascade SMS: page Daniel Taylor (+32 94 953 438); if no ack in 10 minutes, page Miguel Fitzgerald (+32 81 609 125).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a RACI note: Owner Mary Sims; Approver John Graham; Contributors Erin Johnson and Kristy Peterson for customer communications templates.",
    "labels": []
  },
  {
    "text": "Write a profile completeness check for Emily Jones (Bruges; emily.jones@skynet.be; +32 15 726 931). Ask to confirm postal address and marketing preferences.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Compose a consent & preferences audit note for Emily Smith (emily.smith@telenet.be): verify channels against profile (Location Mons; Interest Finance; Hobbies Traveling).",
    "labels": [
      "EMAIL"
    ]
  },
  {
    "text": "Draft a data-quality ticket: Michael Williams shows Gender 'Female' and Age 27; confirm encoding and correct any inconsistencies.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a high-income microsegment (≥ €90k): Katie Lewis (Bruges; €95,193), Jane Miller (Ghent; €93,522), Robert Lewis (Ghent; €99,675).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Compose churn‑risk nudges for Brussels customers: Katie Taylor (katie.taylor@telenet.be), John Johnson (john.johnson@proximus.be), Linda Jones (linda.jones@telenet.be).",
    "labels": [
      "EMAIL"
    ]
  },
  {
    "text": "Write a privacy snippet explaining why we collect IP Address and Interest fields and how customers can manage preferences in the portal.",
    "labels": []
  },
  {
    "text": "Write an engagement snapshot for Emily Jones: Mobile Banking (User, Active; last 2025-04-25) and Current Account (Holder, Active; last 2024-11-11).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Create a relationship matrix for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).",
    "labels": []
  },
  {
    "text": "Produce a microsegment for Mortgage borrowers (Active): John Lewis (last 2025-09-09), Jane Smith (last 2025-06-08), Tom Smith (last 2025-01-20). Suggest a rate alert campaign.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Draft a re‑engagement nudge for Online Banking users now Inactive: John Taylor (last 2025-03-19) and Katie Taylor (last 2025-05-04).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Cross‑sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Offer a mortgage health check and rate simulation.",
    "labels": []
  },
  {
    "text": "Compose a consent/cohort note for Emily Smith: Online Banking (User, Active; last 2025-08-07), Investment Portfolio (Investor, Active; last 2025-07-09), Personal Loan (Borrower, Active; last 2024-10-19).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Create a data‑minimization note: for customer analytics exports (names, emails, phones, addresses, IDs, DOB, IPs, income), use pseudonymous IDs, mask sensitive fields, and restrict raw access to least‑privileged roles.",
    "labels": []
  },
  {
    "text": "Create a data‑minimization note: when sharing employee contacts (phones, emails, CorpKeys) for operational use, limit distribution to least‑privileged DLs and label ‘Internal‑Only’.",
    "labels": []
  }
]